pkgs/applications/misc/octoprint/plugins.nix

```
@@ -206,6 +206,27 @@ in {
     };
   };
 
+  octoprint-ldap = buildPlugin rec {
+    pname = "OctoPrint-LDAP";
+    version = "1.1.0";
+
+    src = fetchFromGitHub {
+      owner = "gillg";
+      repo = pname;
+      rev = "01016c2";
+      sha256 = "0y3v0zpw6pj3gmb7h54402cl2rh0zhv4y31cwfqf1yxhs7bqvzwf";
+    };
+
+    propagatedBuildInputs = with super; [ ldap ];
+
+    meta = with lib; {
+      description = "Octoprint LDAP auth plugin";
+      homepage = "https://github.com/gillg/OctoPrint-LDAP";
+      license = licenses.agpl3;
```

Looks like that the license is `agpl3Only`.

```suggestion
      license = licenses.agpl3Only;
```